src/gui/properties/speedplotview.cpp

```diff
@@ -202,6 +204,65 @@ quint64 SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
+{
+    using Utils::Misc::SizeUnit;
+
+    // table of supposed nice steps for grid marks to get nice looking quarters of scale
+    const double roundingTable[] = {1.2, 1.6, 2, 2.4, 2.8, 3.2, 4, 6, 8};
+
+    struct SplittedValue
+    {
+        double arg;
+        SizeUnit unit;
+        qint64 sizeInBytes() const
+        {
+            return Utils::Misc::sizeInBytes(arg, unit);
+        }
+    };
+
+    SplittedValue getNiceYScale(double value)
+    {
+        if (value == 0.0) return {0, SizeUnit::Byte};
+        if (value <= 12.0) return {12, SizeUnit::Byte};
```

Comment:
seems you don't need this line.

Reply:
If we remove that line, we can get strangely looking labels on low speeds. For example, starting calculation with 5 bytes/s , we will get such formatLabels() output:
for 0 => 0
for 0.25 => 2
for 0.50 => 3
for 0.75 => 5
for 1.00 => 6
For preventing such awkward labels is this "safety barrier" placed here.